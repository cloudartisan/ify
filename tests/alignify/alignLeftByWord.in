if a == 1 and b == 2: print 'good'
if a == 10 and b == 20: print 'good'
