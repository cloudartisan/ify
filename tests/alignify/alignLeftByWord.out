if a == 1 and b == 2: print 'good'
if a == 10and b == 20: print 'good'
